src/Trixi.jl

```diff
@@ -140,16 +140,18 @@ export LaplaceDiffusion2D,
 
 export GradientVariablesPrimitive, GradientVariablesEntropy
 
-export flux, flux_central, flux_lax_friedrichs, flux_hll, flux_hllc, flux_hlle, flux_godunov,
+export flux, flux_central, flux_lax_friedrichs, flux_hll, flux_hll_cn, flux_hllc, flux_hlle, flux_godunov,
```

Comment:
```suggestion
export flux, flux_central, flux_lax_friedrichs, flux_hll, flux_hll_chen_noelle, flux_hllc, flux_hlle, flux_godunov,
```
Since this new HLL-type flux is unique to the shallow water equation, maybe move it closer to the wintermeyer / fjordholm exporting for consistency. This is very minor and I have no strong opinion on this if you want to ignore it. What do you think @sloede ?

Reply:
I could add in the line in which the HR of Chen and Noelle are exported. I just thought it fits best near by `flux_hll` as it is an HLL type with specific speed approximation